reference/supportability.md

```diff
@@ -0,0 +1,159 @@
+## Supportability
+
+The supportability tool collects Mayastor specific information from the cluster using the [kubectl plugin](https://mayastor.gitbook.io/introduction/v/develop/reference/kubectl-plugin) command-line tool. It uses the dump command, which interacts with the Mayastor services to build an archive (ZIP) file that acts as a placeholder for the bundled information.
```

Comment:
Will the URL point to a release branch once it happens?

Reply:
The URL should point to the latest branch. Fixed it.